src/createServer.js

```diff
@@ -1,8 +1,45 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const { pathname } = new URL(req.url, `http://${req.headers.host}`);
+    const normalizedPathWay = pathname.replace('/file', '') || 'index.html';
+    const pathToFile = path.join(__dirname, '..', 'public', normalizedPathWay);
+
+    res.setHeader('content-type', 'text/plain');
```

Comment:
The content type is set to 'text/plain', which might not be appropriate if you are serving HTML files. Consider setting the content type dynamically based on the file extension.